src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTableIntegrationTest.java

```diff
@@ -236,4 +237,31 @@ public void testMoltenOutputWithMultipleAlleles() throws IOException {
         spec.setTrimWhiteSpace(false);
         spec.executeTest("testMoltenOutputWithMultipleAlleles", this);
     }
+
+    @Test
+    public void testNoFieldsSpecified() throws IOException {
+        final File inputFile = new File(getToolTestDataDir(), "extraheaderlinesdeleted_dbsnp_138.snippet.vcf");
+        final File outputFile = createTempFile(getToolTestDataDir(), "noFieldsSpecifiedOutput.table");
+        final File expectedFile = new File(getToolTestDataDir(), "expected.noFieldsSpecified.table");
+
+        final String[] args = new String[] {"--variant", inputFile.getAbsolutePath(),
+                "-O", outputFile.getAbsolutePath()};
+        runCommandLine(args);
+
+        IntegrationTestSpec.assertEqualTextFiles(outputFile, expectedFile);
+    }
+
+    @Test
+    public void testNoFieldsSpecifiedWithSamples() throws IOException {
+        final File inputFile = new File(getToolTestDataDir(), "1000G.phase3.snippet.vcf");
+        final File outputFile = createTempFile(getToolTestDataDir(), "noFieldsSpecifiedWithSamplesOutput.table");
+        final File expectedFile = new File(getToolTestDataDir(), "expected.noFieldsSpecifiedWithSamples.table");
+
+        final String[] args = new String[] {"--variant", inputFile.getAbsolutePath(),
+                "-O", outputFile.getAbsolutePath()};
+        runCommandLine(args);
+
+        IntegrationTestSpec.assertEqualTextFiles(outputFile, expectedFile);
+    }
+
 }
```

Comment:
After reviewing the changes to the tool, I think you should add a third integration test with a VCF that contains declarations for FORMAT fields in its header, but no samples. This will exercise the following case:

```
            if (samples.isEmpty()) {
                genotypeFieldsToTake.clear();
                asGenotypeFieldsToTake.clear();
```

It's a bit weird, but theoretically possible. You should be able to create this by making a copy of your `dbsnp_138.snippet.vcf` file and manually adding in a declaration for a FORMAT field to its header. Note that you can use `git add` to explicitly add new files to a commit.